pkg/actuators/machineset/controller.go

```diff
@@ -37,12 +40,11 @@ const (
 	// This exposes compute information based on the providerSpec input.
 	// This is needed by the autoscaler to foresee upcoming capacity when scaling from zero.
 	// https://github.com/openshift/enhancements/pull/186
-	// cpuKey    = "machine.openshift.io/vCPU"
-	// memoryKey = "machine.openshift.io/memoryMb"
-	//
-	// // TODO: Do we need set GPU ?
-	// gpuKey = "machine.openshift.io/GPU"
-	profileKey = "machine.openshift.io/Profile"
+
+	profileKey = "machine.openshift.io/profile"
```

Comment:
What is a profile?

Reply:
A profile is essentially the instance type/Machine Type. 

For instance: `bx2-4x16` = 4 CPU, 16 GB Ram

Reply:
I don't think we put the profile on for other platforms, so I'm tempted to say we shouldn't here either. CC @elmiko

Reply:
+1

Reply:
Will remove it.